src/lib/molecules/ExtendedInput/index.js

```diff
@@ -257,7 +257,7 @@ const ExtendedInput = forwardRef((props, ref) => {
     );
 
     return (
-        <Tooltip position="bottom" title={showTooltip ? errorText || tooltipText : ''}>
+        <Tooltip position="bottom" title={tooltipText ? tooltipText : ''}>
```

Comment:
Why not just `title={tooltipText}`?
You need to add also the `isVisible` prop for the `Tooltip` component.